certbot/display/util.py

```diff
@@ -253,9 +253,16 @@ def _return_default(self, prompt, default, cli_flag, force_interactive):
         :rtype: bool
 
         """
-        assert_valid_call(prompt, default, cli_flag, force_interactive)
+        # assert_valid_call(prompt, default, cli_flag, force_interactive)
         if self._can_interact(force_interactive):
             return False
+        elif default is None:
```

Comment:
Alternatively, try to interact anyway? I'm worried about false negatives on `_can_interact` where there isn't a cli flag, which will totally break the feature.